storm-core/src/jvm/org/apache/storm/scheduler/SchedulerAssignment.java

```diff
@@ -55,4 +56,6 @@
     public Set<ExecutorDetails> getExecutors();
     
     public Set<WorkerSlot> getSlots();
+
+    public Map<WorkerSlot, Collection<ExecutorDetails>> getSlotToExecutors();
```

Comment:
@d2r should I delete all of the public declarations for all the methods in this class then?